articles/iot-hub/iot-hub-csharp-csharp-getstarted.md

```diff
@@ -196,7 +203,7 @@ In this section, you create a .NET console app that simulates a device that send
             }
         }
    
-    This method sends a new device-to-cloud message every second. The message contains a JSON-serialized object, with the device ID and a randomly generated number to simulate a wind speed sensor.
+    This method sends a new device-to-cloud message every second. The message contains a JSON-serialized object, with the device ID and randomly generated numbers to simulate a wind speed sensor, a temperature sensor and a humidity sensor.
```

Comment:
Minor tweak - missing comma. Should be: "This method sends a new device-to-cloud message every second. The message contains a JSON-serialized object, with the device ID and randomly generated numbers to simulate a wind speed sensor, a temperature sensor, and a humidity sensor. "